cloud/azure/templates/azure_saml_ses/providers.tf

```diff
@@ -1,3 +1,4 @@
 provider "azurerm" {
   features {}
+  subscription_id = "4ef4ae1b-c966-4ac4-9b7c-a837ea410821"
```

Comment:
(question) Is there a reason not to get this from a terraform variable and the civiform*config.sh file. I'm concerned this will get forgotten about and cause issues down the road.

Reply:
I tried using the config variable here and it works! I didn't realize that the providers.tf could access the variables but on retrospect it makes a lot of sense.

Reply:
Wait actually it was because I forgot to push my changes

Reply:
Okay it looks like the providers.tf doesn't have access to the civiform config, but maybe I'm missing something?